integration-test/common/src/test/java/com/tencent/rss/test/HealthCheckCoordinatorGrpcTest.java

```diff
@@ -102,7 +102,7 @@ public void healthCheckTest() throws Exception {
           1,
           1,
           Sets.newHashSet(Constants.SHUFFLE_SERVER_VERSION));
-    Uninterruptibles.sleepUninterruptibly(3, TimeUnit.SECONDS);
+    Uninterruptibles.sleepUninterruptibly(5, TimeUnit.SECONDS);
```

Comment:
is it possible to replace sleep with CoordinatorTestUtils.waitForRegister(coordinatorClient,2)?

Reply:
It's not useful. I try to add `CoordinatorTestUtils.waitForRegister`. But this test is still flaky.

Reply:
here is to test if coordinator has 2 shuffle server
how about using `CoordinatorTestUtils.waitForRegister` to `replace assertEquals(2, coordinatorClient.getShuffleServerList().getServersCount())`;

Reply:
It's not flaky place.

Reply:
We should sleep enough time to receive  the unheathly  heatbeat .

Reply:
got it